### PR TITLE
[SDK] Improve autogenerated stubs

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose

- Exclude *_stubs file from precompiled headers
- Do not include windef.h in stubs.h

This prevents conflicts between incompatible functions declarations in headers and autogenerated stubs.

### CODE REVIEW

**Review by learn-more — APPROVED** [review]

**Review by DarkFire01 — APPROVED** [review]

File: sdk/include/reactos/stubs.h [hunk 1]
```diff
@@ -9,24 +31,30 @@
...
-    _In_z_ _Printf_format_string_ PCSTR Format,
+    const char* Format,
     ...
 );
 
 #define __wine_spec_unimplemented_stub(module, function) \
 { \
     EXCEPTION_RECORD ExceptionRecord = {0}; \
-    ExceptionRecord.ExceptionRecord = NULL; \
+    ExceptionRecord.ExceptionRecord = 0; \
```
Comment (HBelusca) [inline sdk/include/reactos/stubs.h:57]: Which definition of `EXCEPTION_RECORD` structure it uses there? The one with `struct _EXCEPTION_RECORD *ExceptionRecord` member ? If so, why not using `((void *)0)` here?

File: sdk/include/reactos/stubs.h [hunk 2]
```diff
@@ -9,24 +31,30 @@
 #define EXCEPTION_WINE_STUB     0x80000100
 #define EH_NONCONTINUABLE       0x01
 
-NTSYSAPI
-VOID
-NTAPI
+/* __int128 is not supported on x86, so use a custom type */
```
Comment (HBelusca) [inline sdk/include/reactos/stubs.h:34]: Isn't it possible to keep this definition within the spec2def generator, so as to keep "locality" of this MyInt128 thing?
(By the way, see commit bd27b050fcd44d38cfbbe4d054dffecfc46c2432 for why this has been introduced.)

File: sdk/include/reactos/stubs.h (hunk 1 above)
Comment (tkreuzer) [reply sdk/include/reactos/stubs.h:57]: Because 0 is C++ compatible.

File: sdk/include/reactos/stubs.h (hunk 2 above)
Comment (tkreuzer) [reply sdk/include/reactos/stubs.h:34]: Why? The whole point of stubs.h is to provide the definitions for the autogenerated stubs file. And that's it. Yes, it's also used in some manually written stubs files, but hopefully nobody will ever include it into anything else. So why duplicate this into every single stubs file?

Comment (HBelusca) [reply sdk/include/reactos/stubs.h:34]: Then let's go with this, I guess...

**Review by HBelusca — APPROVED** [review]

**Pull request merged**